src/ImageSequencer.js

```diff
@@ -46,6 +46,13 @@ ImageSequencer = function ImageSequencer(options) {
   events = require('./ui/UserInterface')(),
   fs = require('fs');
 
+
+  if(options.inBrowser){
+    for(o in sequencer){
+        modules[o] = sequencer[o];
```

Comment:
do we want to use the `sequencer._____` namespace? what if there is a module name collision with a sequencer property?

Reply:
@jywarren Actually sequencer object is not yet built at this stage hence even if there is a collision that not be a issue. we will consume the property here and later the value of sequencer will be reassigned

Reply:
Ah, maybe I am mistaking this `sequencer` variable for the instance of ImageSequencer used, for example, in the demo. Perhaps this should have a different name here to avoid confusion? Like, in the demos, we say: 

```js
var sequencer = new ImageSequencer(...); // something like this...
```